### ISSUE TITLE
cannot initialise as_quat_array

### ISSUE THREAD

**Issue by mmmatjaz** [moble/quaternion#65]
I'm loading a list of quaternions from a file:
`self.imu=np.loadtxt(fn,skiprows=5)`
then
`self.quat=quaternion.as_quat_array(self.imu[:,11:15])`
spits out an error:
`  File "C:\opt\Anaconda3\lib\site-packages\quaternion\__init__.py", line 87, in as_quat_array
    av = a.view(np.quaternion)

ValueError: To change to a dtype of a different size, the array must be C-contiguous`

I get same result, even if I try just a few rows:
```
array([[ 0.045172, -0.152222, -0.00765 , -0.987284],
       [ 0.045122, -0.152179, -0.007667, -0.987293],
       [ 0.045126, -0.152174, -0.007627, -0.987294],
       [ 0.045131, -0.152172, -0.007567, -0.987294],
       [ 0.045187, -0.152187, -0.007577, -0.987289],
       [ 0.045163, -0.152169, -0.007626, -0.987293],
       [ 0.045164, -0.152174, -0.007664, -0.987292],
       [ 0.045186, -0.152155, -0.007652, -0.987294],
       [ 0.045173, -0.152151, -0.007667, -0.987295]])
```
`self.quat=quaternion.as_quat_array(np.random.rand(10,4))` works fine.

Row by row converts to a quaternion just fine.

**Comment by mmmatjaz** [comment]
workaround:
```
self.imu=np.loadtxt(fn,skiprows=5)
quat=np.zeros((len(self.imu),4))
quat[:,:]=self.imu[:,11:15]
self.quat=quaternion.as_quat_array(quat)
```

**Comment by moble** [comment]
As the error says, this function only works on contiguous data.  The `np.random.rand(10,4)` array is contiguous, as is your original `self.imu` array.  But when you slice it by taking `self.imu[:,11:15]`, numpy returns a non-contiguous array.  In your second comment, you have hit on one way to copy this non-contiguous array into a contiguous one.  A simpler method would be this:

```python
self.imu=np.loadtxt(fn,skiprows=5)
quat = self.imu[:,11:15].copy()
self.quat=quaternion.as_quat_array(quat)
```

Still, this function used to work on contiguous arrays.  It looks like a [pull request](https://github.com/moble/quaternion/pull/50/files) changed it to use an incorrect test of contiguousness.  I'll fix that ASAP.

**Comment by moble** [comment]
Fixed in dee129f0de168b366d7eaf9be632928637415d41.  The conda packages are still being built, but manual installation or pip should work.

**Comment by mmmatjaz** [comment]
Huh, silly me. Thanks for your quick response!
